spec/requests/users_spec.rb

```diff
@@ -1,7 +1,27 @@
 require 'rails_helper'
 
 RSpec.describe 'Users', type: :request do
-  describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+  describe 'GET /users' do
+    it 'returns a successful response' do
+      get users_path
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the index template' do
+      get users_path
+      expect(response).to render_template(:index)
+    end
+  end
+
+  describe 'GET /users/:id' do
+    it 'returns a successful response' do
+      get user_posts_path(User.first)
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the show template' do
+      get user_posts_path(User.first)
+      expect(response).to render_template(:show)
+    end
```

Comment:
- [ ] @Samuelwanza Great work adding tests for the user index and show pages, however, you forgot to include a test that shows  `the response body includes correct placeholder text.`, kindly consider adding it, since it is part of the project requirement.
![image](https://github.com/Samuelwanza/sema/assets/61408860/9cbc3f3d-1d43-4ad4-aeb8-533685068ce8)